### PR TITLE
add context to provide deviceId

### LINKED ISSUE #24: Device Id in Request
It would be nice to provide the deviceId in the EchoRequest (when its enabled in the settings)[amazon dev docs](https://developer.amazon.com/de/docs/custom-skills/device-address-api.html#get-the-consent-token-and-device-id).

We just have to add this to the EchoRequest type. I would create a PR for it.

### PR DESCRIPTION
**Pull request by sholzmayer**
Closes #24

### CODE REVIEW

File: skillserver/echo.go [hunk 1]
```diff
@@ -185,6 +186,14 @@ type EchoSession struct {
 	} `json:"user"`
 }
 
+type EchoContext struct {
+	System struct {
+		Device struct {
+			DeviceId string `json:"deviceId,omitempty"`
```
Comment (rking788) [inline skillserver/echo.go:192]: It would be great to also add the supportedInterfaces property for anyone that wants to build audio or video skills.

Comment (harrisonhjones) [reply skillserver/echo.go:192]: Good catch. We could always add it later if @sholzmayer doesn't want to make the change

**Comment by mikeflynn** [comment]
This change looks good to me and is pretty minimal. Do we want to wait for the supportedInterfaces or just merge this?

**Comment by harrisonhjones** [comment]
I'm happy to just merge it and then I'll add support for `supportedInterfaces` in the refactor

**Comment by rking788** [comment]
agreed, merging sounds fine.

**Comment by mikeflynn** [comment]
Merged! Thanks, @sholzmayer!

**Pull request merged**